small-talk/app/food/page.js

```diff
@@ -9,7 +9,7 @@ export default function food() {
             <div className="flex flex-row">
                 <ul className="m-8 w-full">
                     <li className="border-b border-gray-200 pb-2 mb-10">Order Food</li>
-                    <li className="border-b border-gray-200 pb-2 mb-10">Todays Menu</li>
+                    <li className="border-b border-gray-200 pb-2 mb-10">Today&apos;s Menu</li>
```

Comment:
Was this the error from the linter? Very weird

Reply:
Yea, I originally changed it to "Today's Menu" and the jest test didn't give any errors/warnings, just the linter.

Reply:
Sounds like a pain. We could potentially just ignore that specific rule from the linter